service/src/main/java/bio/terra/pipelines/stairway/imputation/CheckLeonardoHealthStep.java

```diff
@@ -29,7 +29,7 @@ public StepResult doStep(FlightContext flightContext) {
 
   @Override
   public StepResult undoStep(FlightContext flightContext) {
-    // this is the first step in RunImputationJobFlight.
+    // this is the first step in RunImputationAzureJobFlight.
     // increment pipeline failed counter if undoStep is called which means the flight failed
```

Comment:
we should make sure to add a note somewhere for us to increment this metric in th first step of the gcp flight

Reply:
good idea

Reply:
or we implement the webhook ticket, right?

Reply:
I'll make a ticket to add this to the first step of the GCP flight, that's lower lift

Reply:
https://broadworkbench.atlassian.net/browse/TSPS-299